### PR TITLE
New "about" page

### PR DESCRIPTION
**Pull request by nlharris**
This is mostly taken from the "wall of text" on the current home page. The plan is to simplify the text on the home page and move most of the details to this new About page (see #241).
I think it could still use some polishing, but this is a starting point.
Note that at the moment this page is not linked anywhere. Will do that later.

### CODE REVIEW

**Comment by nlharris** [comment]
Actually, I'm not sure it will work to have the md stuff in an html. But I couldn't make this about.md because there is already another page by that name (see #1287).

File: about.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+To achieve this, OBO Foundry participants follow and contribute to the development of an evolving set of <a href="/principles/fp-000-summary.html">principles</a>
+including <a href="/principles/fp-001-open.html">open use</a>, <a href="/principles/fp-010-collaboration.html">collaborative development</a>, <a href="/principles/fp-005-delineated-content.html">non-overlapping and strictly-scoped content</a>, and common <a href="/principles/fp-002-format.html">syntax</a> and <a href="/principles/fp-007-relations.html">relations</a>,
+based on ontology models that work well, such as the <a target="_new" href="http://geneontology.org/>Gene Ontology (GO)</a>.
+       
+<p>The OBO Foundry is overseen by an <a href="http://obofoundry.org/docs/OperationsCommittee.html">Operations Committee</a> with
+<a href="/docs/EditorialWG.html">Editorial</a>, <a href="/docs/TechnicalWG.html">Technical</a> and <a href="/docs/OutreachWG.html">Outreach</a> working groups.</a>
+      
+<p>You can contribute to the OBO Foundry by submitting issues or pull requests to our GitHub repo, <a href="https://github.com/OBOFoundry/OBOFoundry.github.io">OBOFoundry/OBOFoundry.github.io</a>,
```
Comment (nicolevasilevsky) [inline about.html:16]: should we make a bulleted list here to try to avoid a WOT? :) (I love this acronym!)

Comment (nicolevasilevsky) [reply about.html:16]: It's not a very long paragraph though, either way :)

**Review by nicolevasilevsky — APPROVED** [review]

File: about.html (hunk 1 above)
Comment (nlharris) [reply about.html:16]: Let's see how it looks when it's on the web page. I suspect it will need some formatting tweaks.

**Pull request merged**